.travis.yml

```diff
@@ -0,0 +1,38 @@
+language: python
```

Comment:
I'm assuming you've already set up the build yourself?
If so we should get a [status link](https://docs.travis-ci.com/user/status-images/) to it in the README.